_posts/config/2017-05-06-wi-fi-configuration.md

```diff
@@ -108,9 +108,9 @@ If you select one of these access points, respective wireless controls (i.e., _N
 
 ## Wi-Fi Linux Configuration
 
-This section describes the changes applied by Kura at the Linux networking configuration. Please read the following note before proceeding with manual changes of the Linux networking configuration.
+This section describes the changes applied by Kura at the Linux networking configuration. Please read the following note before proceeding with manual changes in the Linux networking configuration.
 
-{% include alerts.html message='It is NOT recommended performing manual editing of the Linux networking configuration files when the gateway configuration is being managed through Kura. While Linux may correctly accept manual changes, Kura may not be able to interpret the new configuration resulting in an inconsistent state.' %}
+{% include alerts.html message='It is NOT recommended to perform manual editing of the Linux networking configuration files when the gateway configuration is being managed through Kura. While Linux may correctly accept manual changes, Kura may not be able to interpret the new configuration resulting in an inconsistent state.' %}
```

Comment:
This whole paragraph should be changed to:
"Manual editing of the Linux networking configuration file is NOT recommended when the gateway configuration is being managed through Kura."